receiver/sshcheckreceiver/README.md

```diff
@@ -0,0 +1,46 @@
+# SSH Check Receiver
+
+| Status                   |           |
+| ------------------------ |-----------|
+| Stability                | [beta]    |
+| Supported pipeline types | metrics   |
+| Distributions            | [contrib] |
+
+This receiver creates stats by connecting to an SSH server which may be an SFTP server.
+
+## Prerequisites
+
+If `IgnoreHostKey` is not set then host key validation requires the agent either have a known_hosts file at $HOME/.ssh/known_hosts or /etc/ssh/known_hosts.
```

Comment:
Maybe mention the config value instead, i.e. `ignore_host_key`? Also, worth mentioning it in the Configuration section below.